frontend/javascripts/oxalis/model/sagas/isosurface_saga.ts

```diff
@@ -687,16 +693,22 @@ function* loadPrecomputedMeshForSegmentId(
         segmentInfo.transform[1][1],
         segmentInfo.transform[2][2],
       ];
-      availableChunks = _.first(segmentInfo.chunks.lods)?.chunks || [];
+      segmentInfo.chunks.lods.forEach((chunks, lodIndex) => {
+        availableChunksMap[lodIndex] = chunks?.chunks;
+        loadingOrder.push(lodIndex);
+      });
+      // Load the chunks closest to the current LOD first.
+      loadingOrder.sort((a, b) => Math.abs(a - currentLODIndex) - Math.abs(b - currentLODIndex));
```

Comment:
see my other comment about some sort of progressive rendering. if this **would** be possible, I'd prefer to load the coarsest lod first. if it's not possible, it's probably best to keep it as is.

Reply:
I don't think that's possible in an easy way. This would likely need a more sophisticated approach to calculate which segments overlap / cover each other in different LODs

Reply:
Ok, then let's keep it as is for this iteration :)